azkaban-exec-server/src/main/java/azkaban/container/FlowContainer.java

```diff
@@ -111,10 +124,16 @@
   private final ExecutorLoader executorLoader;
   private final ProjectLoader projectLoader;
   private final JobTypeManager jobTypeManager;
+  private final Server jettyServer;
+  private final Context containerContext;
+  private final AzkabanEventReporter eventReporter;
   private final Props azKabanProps;
   private Props globalProps;
   private final int numJobThreadPerFlow;
   private String execDirPath;
+  private int port; // Listener port for incoming control & log messages (ContainerServlet)
```

Comment:
FlowContainer is expected to listen on a fixed local port, which I don't think happens currently. (Reverse-proxy settings need this to be fixed)

Would be ok to do it in a separate PR, please create a TODO for it in that case.